doc/deployment/deploy_custom.md

```diff
@@ -84,203 +43,219 @@ pachctl deploy custom --persistent-disk <persistent disk backend> --object-store
+Support for S3v2 is being deprecated. Therefore Pachyderm recommends using S3v4 in all deployments.
 
 If you need to access an object store that uses S3v2 signatures,
-you can specify the flag `--isS3V2`. 
+you can specify the `--isS3V2` flag.
 
-This will configure Pachyderm to use the Min.io driver,
+This parameter configures Pachyderm to use the MinIO driver,
 allowing the use of the older signature.
-Using this flag will also disable SSL for connections to the object store with the `minio` driver.
+Also, this flag disables SSL for connections to the object store with the `minio` driver.
 You can reenable it with the `-s` or `--secure` flag.
-You can also edit the `pachyderm-storage-secret` Kubernetes manifest it produces manually,
-which is detailed below.
+You can also edit the `pachyderm-storage-secret` Kubernetes manifest,
+as described below.
 
 #### Object store parameters
 
 The `--object-store` flag takes four (4) required, additional configuration arguments.
 These arguments must be placed immediately after [the persistent disk configuration parameters](#persistent-disk-parameters).
 
-- _bucket-name_: the name of the bucket, without the `s3://` prefix or a trailing `/`.
-- _access-key_: the user access id used to access the object store.
-- _secret-key_: the associated password used with the user access id to access the object store.
-- _endpoint_: the hostname and port used to access the object store, in <hostname>:<port> format.
+- _bucket-name_: The name of the bucket, without the `s3://` prefix or a trailing forward slash (`/`).
+- _access-key_: The user access ID that is used to access the object store.
+- _secret-key_: The associated password that is used with the user access id to access the object store.
+- _endpoint_: The hostname and port that are used to access the object store, in <hostname>:<port> format.
 
 #### Example invocation with pv and object store
-Our example on-premises cluster will use an on-premises Min.io object store
-  - with SSL turned on,
+The example on-premises cluster will use an on-premises MinIO object store with the following configuration parameters:
+  - SSL turned on,
   - S3v4 signatures,
   - the endpoint `minio:9000`
   - the access key `OBSIJRBE0PP2NO4QOA27`,
   - the secret key `tfteSlswRu7BJ86wekitnifILbZam1KYY3TG`,
   - and a bucket named `pachyderm-bucket`.
-Our deployment flags for our sample cluster look like this, so far:
+The deployment command uses the following flags:
 ```
 pachctl deploy custom --persistent-disk aws --object-store s3 \
     foobar 10 \
     pachyderm-bucket  'OBSIJRBE0PP2NO4QOA27' 'tfteSlswRu7BJ86wekitnifILbZam1KYY3TG' 'minio:9000' \
     --dynamic-etcd-nodes 10
     [optional flags]
 ```
-Note that we enclosed the arguments that may contain characters that the shell could interpret in single-quotes.
+In the example command above, some of the arguments may contain characters that the shell could interpret in single-quotes.
```

Comment:
"could interpret in single quotes"? What does this mean. Does it mean I need single quotes, double quotes, on whic arguments?